samples_project/Assets/SampleViewer/Samples/Measure/Measure.cs

```diff
@@ -101,28 +103,31 @@ private void Interpolate(GameObject start, GameObject end, List<GameObject> feat
         double d = ArcGISGeometryEngine.DistanceGeodetic(startPoint, endPoint, currentUnit, new ArcGISAngularUnit(ArcGISAngularUnitId.Degrees), ArcGISGeodeticCurveType.Geodesic).Distance;
         float n = Mathf.Floor((float)d / InterpolationInterval);
         double dx = (end.transform.position.x - start.transform.position.x) / n;
+        double dy = (end.transform.position.y - start.transform.position.y) / n;
         double dz = (end.transform.position.z - start.transform.position.z) / n;
 
-        var pre = start.transform.position;
+        var PreviousInterpolation = start.transform.position;
 
         // Calculate n-1 intepolation points/n-1 segments because the last segment is already created by the end point.
         for (int i = 0; i < n - 1; i++)
         {
-            GameObject next = Instantiate(InterpolationMarker, arcGISMapComponent.transform);
+            GameObject NextInterpolation = Instantiate(InterpolationMarker, arcGISMapComponent.transform);
```

Comment:
```suggestion
            GameObject nextInterpolation = Instantiate(InterpolationMarker, arcGISMapComponent.transform);
```
Should be camelcase for local variable in C#.